packages/fxa-payments-server/src/components/PaymentForm/index.test.tsx

```diff
@@ -148,20 +154,26 @@ it('when confirm = true, enables submit button when all fields are valid and che
   expect(getByTestId('submit')).toHaveClass('payment-button-disabled');
   fireEvent.click(getByTestId('confirm'));
   expect(getByTestId('submit')).not.toHaveClass('payment-button-disabled');
+  expect(ReactGALog.logEvent).toBeCalledTimes(1);
+  expect(ReactGALog.logEvent).toBeCalledWith(
+    MOCK_EVENTS.AddPaymentInfo(SELECTED_PLAN)
+  );
 });
 
 it('omits the confirmation checkbox when confirm = false', () => {
   const { queryByTestId } = renderWithLocalizationProvider(
     <Subject {...{ confirm: false }} />
   );
   expect(queryByTestId('confirm')).not.toBeInTheDocument();
+  expect(ReactGALog.logEvent).toBeCalledTimes(0);
```

Comment:
Interestin'. Doesn't look like we have a strong convention between `.toBeCalledTimes(0)` versus `.not.toBeCalled()`:

| CMD | `wc -l`
:----|----:|
<kbd>git grep ".toBeCalledTimes(0)" packages/fxa-payments-server</kbd>  | 5
<kbd>git grep ".toHaveBeenCalledTimes(0)" packages/fxa-payments-server</kbd>  | 14
<kbd>git grep ".not.toHaveBeenCalled()" packages/fxa-payments-server</kbd>  |  53
<kbd>git grep ".not.toBeCalled()" packages/fxa-payments-server</kbd>  |  6

Reply:
I observed this too and decided to just be consistent in how I wrote it in all my tests. 👀

Reply:
Good catch. I thought of this more as a contextual thing, but is there very much of a difference? I have found use cases for both and I'm not sure where there is a specific difference, author's choice, or legacy use cases in our code.

Reply:
https://github.com/jestjs/jest/blob/0fd5b1c37555f485c56a6ad2d6b010a72204f9f6/packages/expect/src/spyMatchers.ts#L357

If I'm looking at the correct code, `.toBeCalled()` matcher is just checking `const pass = count > 0`, so they should do the same thing, just more... concise-er?

But looks like fxa-payments-server uses both methods and both aliases. Although tiniest of nits and not worth the time spent refactoring.
I'd probably lean towards `.not.toBeCalled()` for the zero case, and `.toBeCalledTimes(n)` for non-zero cases where we specify the exact number of times the function was called. Using `.toBeCalled()` without the negation seems like it _could_ be risky since we wouldn't know if the function was called once or an unexpected 11 times.

```sh
git grep -Pn "\.to(Be|HaveBeen)Called\(\)" | grep -v ").not.to" | wc -l # 74
```